pkg/quarterdeck/report/report.go

```diff
@@ -255,13 +255,15 @@ func (r *DailyUsers) NewAccounts(tx *sql.Tx, day sql.NamedArg) (accounts []*emai
 			napikeys  sql.NullInt64
 			nusers    sql.NullInt64
 			ninvites  sql.NullInt64
+			lastLogin sql.NullString
 		)
 
 		a := &emails.NewAccountData{}
-		if err = rows.Scan(&a.Name, &a.Email, &a.EmailVerified, &a.Role, &a.LastLogin, &a.Created, &a.Organization, &a.Domain, &nprojects, &napikeys, &nusers, &ninvites); err != nil {
+		if err = rows.Scan(&a.Name, &a.Email, &a.EmailVerified, &a.Role, &lastLogin, &a.Created, &a.Organization, &a.Domain, &nprojects, &napikeys, &nusers, &ninvites); err != nil {
```

Comment:
name, email, created, organization, and domain are all NOT NULL in the database. EmailVerified defaults to false, so shouldn't be null ever. The question is about role … this is a foreign key item so I don't think it should be null either.